Formula/osquery.rb

```diff
@@ -183,8 +183,14 @@ def install
       -Dcppnetlib-uri_library:FILEPATH=#{vendor}/cpp-netlib/lib/libcppnetlib-uri.a
       -Dlinenoise_library:FILEPATH=#{vendor}/linenoise/lib/liblinenoise.a
       -Dthrift_library:FILEPATH=#{vendor}/thrift/lib/libthrift.a
-      -DCMAKE_CXX_FLAGS_RELEASE:STRING=-DNDEBUG\ -I#{MacOS.sdk_path}/usr/include/libxml2\ -I#{vendor}/aws-sdk-cpp/include\ -I#{vendor}/cpp-netlib/include\ -I#{vendor}/linenoise/include\ -I#{vendor}/thrift/include\ -Wl,-L#{vendor}/linenoise/lib
     ]
+    args << "-DCMAKE_CXX_FLAGS_RELEASE:STRING=-DNDEBUG\ "\
```

Comment:
Having just one outside of the array seems really weird.

Reply:
The way it's being done at all seems weird. We could set it as a variable first or something and interpolate if that would be better.

Reply:
yeah maybe like this https://github.com/Homebrew/homebrew-core/blob/master/Formula/acmetool.rb#L163-L169